src/site/content/en/blog/storage-for-the-web/index.md

```diff
@@ -159,7 +163,14 @@ of stored cross origin resources.
 During development, you can use your browser's DevTools to inspect the
 different storage types, and easily clear all stored data.
 
-<img class="w-screenshot w-screenshot-filled" alt="Storage test tool." src="storage-test-tool.png">
+A new feature was added in Chrome 88 that lets you override the site's storage
+quota in the Storage Pane. This feature gives you the ability to simulate
+different devices and test the behavior of your apps in low disk availability
+scenarios. Go to **Application** then **Storage**, enable the
+**Simulate custom storage quota** checkbox, and enter any valid number to
+simulate the storage quota.
+
+<img class="w-screenshot w-screenshot-filled" alt="DevTools Storage pane." src="devtools-storage-panel.png">
```

Comment:
This change seems fine but could you upload the image via the new [flow](https://web.dev/handbook/markup-media/)?

Reply:
Done.